jaeger-zipkin/src/main/java/com/uber/jaeger/senders/zipkin/ZipkinSender.java

```diff
@@ -125,11 +124,6 @@ public int append(Span span) throws SenderException {
     return n;
```

Comment:
it is because the additional value changes the size. the new sender works with scribe, which doesn't have a constant size function based on span count (since each line is base64 and depending equals counts, this varies). in order to be a stateless function, it has to recalculate the size. benchmarking shows this as a trivial amount of cycles.